tools/fetch.py

```diff
@@ -156,9 +159,9 @@ def vx_start(nodelist,outdir,cmd, timeout=0, threadcount=20):
     parser.add_option("", "--external", dest="external",  action="store_true",
                         help=("Run commands external to the server. The "+
                               "default is internal."))
-    parser.add_option("", "--template", dest="template", 
-                        help=("Command template for external commands; "+
-                              "substitutes [] with hostname."))
+    parser.add_option(
+        "", "--user", dest="user", metavar="username", default="site_admin",
```

Comment:
I agree this is inconsistent. This file needs a style clean up. After this change, the next one will address all style issues in this file. Using the default= argument to add_option is preferable to set_defaults(). So, the next revision will eliminate the set_defaults call.

Sound alright?